replication-api/src/main/java/org/codice/ditto/replication/api/ReplicationItemManager.java

```diff
@@ -17,9 +17,9 @@
 import java.util.Optional;
 import org.codice.ddf.persistence.PersistenceException;
 
-public interface ReplicationPersistentStore {
+public interface ReplicationItemManager {
 
-  Optional<ReplicationItem> getItem(String id, String source, String destination);
+  Optional<ReplicationItem> getItem(String metacardId, String source, String destination);
 
   List<ReplicationItem> getItemsForConfig(String configId, int startIndex, int pageSize)
```

Comment:
❗️ That start index is concerning. Are we guaranteed that items are always returned in the same order on every queries and that new ones are only added at the end? What about if some are deleted, wouldn't that starting index be impacted and suddenly miss or skip items?

Reply:
I had similar concerns. I haven't investigated it too much yet though but I will take a closer look.

Reply:
Pretty sure the index and page size are passed directly on to solr so as long as it is doing it's job we are ok. When we switch out the backend we will need to make sure that is still the case with whatever we use.